.rubocop_todo.yml

```diff
@@ -33,6 +40,13 @@ Rails/HelperInstanceVariable:
   Exclude:
     - 'app/helpers/application_helper.rb'
 
+# Offense count: 1
+# Configuration parameters: Include.
+# Include: app/**/*.rb, config/**/*.rb, db/**/*.rb, lib/**/*.rb
+Rails/Output:
+  Exclude:
+    - 'app/services/m.rb'
```

Comment:
what's `m.rb`?

Reply:
oh that's a custom file I use with pry but I'd forgotten about it. Let me remove this.